CHANGELOG.md

```diff
@@ -17,6 +17,7 @@
 - [\#467](https://github.com/cosmos/evm/pull/467) Replace GlobalEVMMempool by passing to JSONRPC on initiate.
 - [\#352](https://github.com/cosmos/evm/pull/352) Remove the creation of a Geth EVM instance, stateDB during the AnteHandler balance check.
 - [\#496](https://github.com/cosmos/evm/pull/496) Simplify mempool instantiation by using configs instead of objects.
+- [\#535](https://github.com/cosmos/evm/pull/535) Add nested precompile reversion tests.
```

Comment:
imo, adding a test doesn't warrant a changelog entry, since this doesn't really alter the offering of the software

Reply:
Good call, will revert this and wait on the merge for the removal of the limit.